enhancements/single-node/aos-sno-pair.md

```diff
@@ -0,0 +1,143 @@
+
+For customers that do have singleton workloads, the RHEL-HA stack supports a wide variety of BMCs including those conforming to the IPMI and RedFish standards, as well as SSH and libvirt for testing.
+
+It also supports indirect fencing methods like poison-pill which can optionally make use of shared storage for out-of-band signalling and heartbeating where available.
+
+#### Quorum handling
+_The "two node cluster" is a use case that requires special consideration.  With a standard two node cluster, each node with a single vote, there are 2 votes in the cluster. Using the simple majority calculation (50% of the votes + 1)  to calculate quorum, the quorum would be 2.  This means that the both nodes would always have to be alive for the cluster to be quorate and operate._
+<br>Source: man votequorum
+
+For environments in which customers will allow it, Corosync also ships a lightweight quorum arbitrator that can be run locally or in the cloud on any non-cluster node.  In such cases, no special quorum handling is required and careful placement of the arbitrator also ensures that the surviving peer is reachable by its intended clients.
+
+Where this is not possible, there are existing capabilities that we can leverage by enabling the _Corosync two_node, last_man_standing, and wait_for_all_ options (See: man votequorum) and configuring a fencing delay in Pacemaker.
+
+In combination, these options provide the following behaviour:
+- Both sides retain quorum in the event of a network split.
+- Service recovery is normally predicated on successful fencing.
+- When the fenced node reboots, it will not be able to obtain quorum until it has seen its peer (preventing an infinite fencing cycle).
+- To avoid both sides fencing each other at the same time, fencing is typically configured to occur after a delay on one of the nodes. That node will perform recovery in the event of a true peer failure, but will always lose in the response to a pure network failure.
+
+### Risks and Mitigations
+- A two node etcd cluster does not have quorum whenever the communication between the peers is interrupted (by network or peer failure).
+- From inside a typical 2 node cluster, network and peer failures look identical.
+
+## Design Details
+
+### Open Questions
+
+- Do we really need active/active, or could active/passive be sufficient?
+  - Reasons mentioned in favor:
+    - It ensures the secondary node works when you need it. Not only it is working, but the whole list of dependencies it carries with it (DNS...)
+    - Performance.
+    - Cost. The secondary node is not just sitting there, wasting resources.
+    - LESS maintenance procedures. You don't need to failover/failback on upgrades, for example.
+- Do we really need automated failover, or is manual confirmation sufficient?
+- How much overlap is there between the goals as laid out here and Telco RAN requirements?
+
+
+### Test Plan
+
+### Graduation Criteria
```

Comment:
Are we going to release this in a preview state first or go straight to GA?

Reply:
I don't think graduating to GA is assumed at this point.
At the moment, we're just looking to socialize the concept prior to putting a demo in the hands of customers for feedback.  There are still issues around config and data synchronization that we need to have a story for before this is something we can GA.

Reply:
Listing those items as requirements for GA is exactly the sort of detail we should capture early, so it isn't lost.

Reply:
I've added a template on how we plan to fill GA Criteria.
We will fill in more concrete details based on milestones progress.